pkg/network/node/vxlan_monitor_test.go

```diff
@@ -67,7 +67,7 @@ func TestEgressVXLANMonitor(t *testing.T) {
 
 	// Everything should be fine at startup
 	retry := evm.check(false)
-	if update := peekUpdate(updates); update != nil {
+	if update := peekUpdate(updates, evm); update != nil {
 		t.Fatalf("Initial check showed updated node %#v", update)
```

Comment:
while you're updating this anyway, you changed "node" to "nodes" in about half of the error messages here, but not the other half...